content/articles/serialization-in-csharp/index.md

```diff
@@ -0,0 +1,169 @@
+### Introduction
+Serialization transforms a data structure or object state into a format that can be saved in a file, memory or sent over a computer network and then reconstructed. This is a benefit of Serialization in terms of transmitting the converted data.
+
+### Preliquisites
+- Have [visual studio 2019](https://visualstudio.microsoft.com/vs/) software installed. It is the compiler used to run Csharp programs.
+- Have some prior knowledge in .NET programming and xml files.
+
+### Table of contents
+- [Serialization applications](#serialization-applications)
+- [Making a serializable object](#making-a-serializable-object)
+- [C# Serialization in action](#c#-serialization-in-action)
+- [Types of serialization in C#](Types-of-serialization-in-#c#)
+- [Conclusion](#conclusion)
+
+### Serialization applications
+Serialization allows developers to save an object's state and restore it later if needed. Serialization allows the following operations.
+
+- The item is delivered to the remote application through HTTP.
+- Domain to domain transfer - A JSON or XML data string can be used to transport an item over a firewall.

```

Comment:
Paraphrase the first list item to align with the intro sentence such that the two can be read without loss of flow.